src/django/api/gazetteer.py

```diff
@@ -198,9 +178,9 @@ def build_index_table(self, canonical):
         table_name = 'dedupe_indexed_records_{}'.format(self.trained_model.id)
 
         with connection.cursor() as cursor:
-            self._create_index_table(cursor)
             cursor.execute(
-                "CREATE TABLE {} (LIKE dedupe_indexed_records INCLUDING ALL)"
+                "CREATE TABLE {} (LIKE dedupe_indexed_records INCLUDING "
+                                  "ALL)"
                 .format(table_name)
```

Comment:
The linter is unhappy about this
```
./api/gazetteer.py:183:35: E131 continuation line unaligned for hanging indent
```

 This works 

```suggestion
                "CREATE TABLE {} (LIKE dedupe_indexed_records INCLUDING ALL)"
                .format(table_name)
```